src/components/dashboard/navAdmin/NavAdmin.jsx

```diff
@@ -1,16 +1,17 @@
-import { useSelector } from "react-redux"
-import { Link } from "react-router-dom"
+import { Link } from "react-router-dom";
 import { 
   ListItemButton,
   ListItemIcon,
   ListItemText,
   Tooltip,
 } from '@mui/material';
 import AdminPanelSettingsIcon from '@mui/icons-material/AdminPanelSettings';
+import useGetUserInfo from "../../../hooks/useGetUserInfo";
```

Comment:
Consider simplifying the import paths.

When using CRA, adding env var `NODE_PATH=src/` should then allow simpler `import useGetUserInfo from "hooks/useGetUserInfo";
`

Reply:
Another way is well written up on https://saurabhshah23.medium.com/react-app-with-absolute-paths-using-jsconfig-json-2b07b1cb24d4

Reply:
I'm really happy that you've pointed that out to, never even crossed my mind to see if there was better option.

Reply:
Cool. Lots of relative paths can be a lot of extra text and especially when deeply nesting folders.